cairo/ethereum/utils/numeric.cairo

```diff
@@ -170,6 +170,26 @@ func U256_from_be_bytes20{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(bytes20
     return res;
 }
 
+func U256_to_be_bytes20{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(value: U256) -> Bytes20 {
+    let high = value.value.high;
+    let is_within_bounds = is_le(high, 2 ** 32 - 1);
+    with_attr error_message("OverflowError") {
+        assert is_within_bounds = 1;
+    }
+    let (high_reversed) = word_reverse_endian(high);
+    let (low_reversed) = word_reverse_endian(value.value.low);
+    let (high_reversed_shifted, _) = divmod(high_reversed, 2 ** 96);
+    let res_felt = high_reversed_shifted + low_reversed * 2 ** 32;
+    tempvar res = Bytes20(res_felt);
+    return res;
+}
```

Comment:
This function should be named `U256_to_le_bytes20`. By default all values are expressed in little-endian form, _execpt_ values taken from the stack.

This works in the flow of Address_from_felt_be because the input u256 is big-endian, this the output of this function is a little-endian, instead of big-endian